dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/UnpackagedTests/MrtCoreUnpackagedTests.csproj

```diff
@@ -7,7 +7,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>MrtCoreUnpackagedTests</RootNamespace>
     <AssemblyName>MrtCoreUnpackagedTests</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```

Comment:
This change is coming from main (through a merge) and thus, it shouldn't show up here.

Reply:
Solved